pkg/proxy/ipvs/proxier.go

```diff
@@ -1104,6 +1104,10 @@ func (proxier *Proxier) syncProxyRules() {
 				IP2:      epIP,
 				SetType:  utilipset.HashIPPortIP,
 			}
+			if valid, err := proxier.loopbackSet.validateEntry(entry); !valid {
+				glog.Errorf("Failed to add entry %v to set %s since it's invalid, error: %v", entry, proxier.loopbackSet.Name, err)
```

Comment:
nit: given that this string is constant (and copied throughout) perhaps make it a constant in this file?

Reply:
Sounds fair, I will try. However, the error string contains some variables and the variables will break the complete string. For instance,

```
const EntryInvalidErr = "entry is invalid"

glog.Error("Failed to add entry %v to set %s, %s, error: %v", entry, set, EntryInvalidErr, err)

```

Reply:
Applied, PTAL.